src/app/qgshandlebadlayers.cpp

```diff
@@ -524,6 +524,7 @@ QString QgsHandleBadLayers::checkBasepath( const QString &layerId, const QString
 
 void QgsHandleBadLayers::autoFind()
 {
+  QgsTaskManager *manager = new QgsTaskManager();
```

Comment:
Don't create a new task manager, instead use the global application one via QgsApplication::taskManager()

Reply:
Forgot about that, Thanks,